.github/workflows/wheels.yml

```
@@ -0,0 +1,43 @@
+name: Build-wheels
```

Our convention (for now) is to use a capitalized noun for the name of each workflow.
```suggestion
name: Wheels
```